CHANGELOG.md

```diff
@@ -1,3 +1,74 @@
+## Fleet 4.55.0 (Aug 8, 2024)
+
+**NOTE:** Beginning with v4.55.0, Fleet no longer supports MySQL 5.7 because it has reached [end of life](https://mattermost.com/blog/mysql-5-7-reached-eol-upgrade-to-mysql-8-x-today/#:~:text=In%20October%202023%2C%20MySQL%205.7,to%20upgrade%20to%20MySQL%208.). The minimum version supported is MySQL 8.0.36.
+
+**NOTE:** Changes to [software field](https://github.com/fleetdm/fleet-gitops/blob/main/teams/workstations.yml#L24) in GitOps:
```

Comment:
```suggestion
**NOTE:** Changes to software field in [GitOps](https://fleetdm.com/docs/using-fleet/gitops):
```

I think let's link to the reference docs. When users/customers see the change live in the docs they're usually like "oh, cool it's documented"